src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_x509.h

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 #include "pal_jni.h"
+#include <pal_x509_types.h>
```

Comment:
```suggestion
#include "pal_x509_types.h"
```

Quotes are local-first, right?

Reply:
I had this because I put `pal_x509_types.h` is under Common, not in this folder?

Reply:
Ah, I'd missed that it was a different directory.  Nevermind.